src/main/webapp/js/instructorFeedbackEdit.js

```diff
@@ -3,6 +3,11 @@ $(document).ready(function() {
     bindUncommonSettingsEvents();
     updateUncommonSettingsInfo();
     hideUncommonPanels();
+
+    $('.dropdown-menu li a').click(function() {
+        $('#questiontype').val(this.id);
```

Comment:
This line should go into `showNewQuestionFrame`, since it seems like some code in `instructorFeedbacks.js` is always relying on this to be set when adding a new question